README.md

```diff
@@ -43,3 +44,19 @@ mount: cannot mount tmpfs read-only
 ```
 
 If you get this tmpfs mount error, add `--security-opt apparmor:unconfined \` to your list of run options. This error has been seen on Ubuntu, but may occur on other platforms as well.
+
+## Changelog
+
+### 2018.02.27
```

Comment:
Generally, I try and have values in the changelog tightly correlate to releases and tags. For example, if you push a github tag, have that be reflected in the changelog.

Reply:
I agree, but I'm having trouble thinking of a way to do that well.

For example, updating to 3.9.2 is an obvious one. I'd put that as `## 3.9.2` right? But in this recent example, we updated to 3.9.2 and included some minor changes. But then afterward, I added some bigger changes on top and *then* tagged it `3.9.2`. Does this all roll up to 3.9.2? In this case, I think it does because of the narrow time frame.

But what if I make some improvements in a week or two. Then a week or two later, 3.9.3 comes out and we update to that. Where do those changes go? :/

Reply:
This was why I suggested not using the matching Unifi Video versions.

Stick with Semver for this repo. I.e. tag somethign 1.0.0. Have the release notes include which unifi version that includes. Then later, if you add patch level changes, it's 1.0.1, but if you upgrade to a version of unifi-video that has breaking changes (or the dockerfile, etc) that would be 2.0.0. The change log would be where you show which version of Unifi-Video you use.

Reply:
https://semver.org/

Reply:
Dang, that makes a lot of sense actually. How about a variant of that like Linux distro's use?

We call it 3.9.2-1 for example. Then 3.9.2-2 when we make changes that aren't unifi-video related. When 3.9.3 comes out, it resets back to 3.9.3-1. This gets the unifi-video version in, which I think is actually pretty important... and we get something to use in the change log. This is how AUR for Arch Linux does things. (And every other Linux too, basically). Reasonable?

Reply:
As long as you have a way of representing things that doesn't make things more confusing I really don't mind :)